### PR TITLE
Add spanHost field into tcollector submit

### PR DESCRIPTION
**Pull request by junchaowu**
Based on Yuri's request adding the spanHost field.
@abhinav @blampe @breerly 

cc @prashantv  @Raynos

### CODE REVIEW

**Comment by blampe** [comment]
@yurishkuro fyi

File: tchannel/zipkin/tracers.py [hunk 1]
```diff
@@ -148,10 +153,18 @@ def submit_callback(f):
                     exc_info=f.exc_info()
                 )
 
+        span_host = self.parse_host_port()
```
Comment (yurishkuro) [inline tchannel/zipkin/tracers.py]: what's the reason not to do it in `__init__`?  We already have tchannel by then.

File: tchannel/zipkin/formatters.py [hunk 2]
```diff
@@ -160,7 +167,9 @@ def thrift_formatter(trace, annotations, isbased64=False):
         host=host,
         parentId=i64_to_string(trace.parent_span_id),
         annotations=thrift_annotations,
-        binaryAnnotations=binary_annotations
+        binaryAnnotations=binary_annotations,
+        debug=False,
```
Comment (yurishkuro) [inline tchannel/zipkin/formatters.py]: a different issue - we should be carrying over the debug flag from the parent span. But it's not critical.

**Comment by yurishkuro** [comment]
lgtm except for one possible performance issue, there's no need to recreate span_host every time

File: tchannel/zipkin/tracers.py (hunk 1 above)
Comment (junchaowu) [inline tchannel/zipkin/tracers.py]: the zipkin trace hook is added usually before tchannel starts listening. Therefore the tchannel host port is not ready at __init__.

**Pull request merged**